docs/adr/AsynkronKoeForBrevBestillinger.md

```diff
@@ -0,0 +1,60 @@
+# Asynkron kø for brevbestillinger
+
+* Status: [draft]
+* Deciders: Håkon Heggholmen
+
+Technical Story: [Lage asynkron kø for brevbestillinger]
+
+## Context and Problem Statement
+
+I dag har vi kun ett synkront løp for bestilling av brev. Det å rendre en PDF med LaTeX tar flere sekunder, og vi har 
+begrenset med ledig synkron kapasitet til en hver tid. For å komme rundt dette problemet i dag må konsumenter hensynta
+våres ledige kapasitet. Konsumenter må også ta stilling til tekniske feil som team pensjonsbrev er ansvarlige for å løse
```

Comment:
Det er ikke nødvendigvis vårt team sitt ansvar å ha en løsning som gjør det lettere å behandle store mengder brev, og redusere sjansen for at requests timer-ut eller at kapasiteten er makset ut under skalering. Det er heller et ansvar vi kan påta oss for å gjøre jobben lettere for andre.
Så kanskje "Konsumenter må selv ta ansvar for å lage løsninger som ikke overbelaster kapasiteten til brevbaker ved å ha en fornuftig algoritme for throttling av requests og retry ved server feil (5xx)".